src/core/types.ts

```diff
@@ -126,3 +126,45 @@ export type FrameMetadataType = {
  * Note: exported as public Type
  */
 export type FrameMetadataResponse = Record<string, string>;
+
+/**
+ * Ethereum Attestation Service (EAS) Attester Address
+ * The Ethereum address of the attester who created the attestation.
+ */
+type EASAttesterAddress = Address;
+
+/**
+ * Ethereum Attestation Service (EAS) Schema Uid
+ * The schema identifier associated with the EAS attestation.
+ * Note: exported as public Type
+ */
+export type EASSchemaUid = `0x${string}`;
+
+/**
+ * Ethereum Attestation Service (EAS) Attestation
+ * GraphQL response for EAS Attestation
+ * Note: exported as public Type
+ */
+export type EASAttestation = {
+  id: string; // The unique identifier of the attestation.
+  decodedDataJson: string; // The attestation data decoded to JSON.
+  recipient: Address; // The Ethereum address of the recipient of the attestation.
+  attester: EASAttesterAddress; // the attester who created the attestation.
+  time: number; // The Unix timestamp when the attestation was created.
+  expirationTime: number; // The Unix timestamp when the attestation expires (0 for no expiration).
+  revocationTime: number; // The Unix timestamp when the attestation was revoked, if applicable.
+  revoked: boolean; // A boolean indicating whether the attestation is revocable or not.
+  schemaId: EASSchemaUid; // The schema identifier associated with the attestation.
+};
+
+/**
+ * Ethereum Attestation Service (EAS) Chain Definition
+ * The definition of a blockchain chain supported by EAS attestations.
+ * Note: exported as public Type
+ */
+export type EASChainDefinition = {
+  id: number; // blockchain source id
+  easGraphqlAPI: string; // EAS GraphQL API endpoint
+  schemaUids: EASSchemaUid[];
+  attesterAddresses: EASAttesterAddress[];
```

Comment:
nit probably an edge case or something to worry about in the future, but a user may not always want all attestations for a given recipient from all attesters on that chain. 

ex: Gitcoin and Coinbase both make attestations against a generalized {verified: true} schema on Base. as a user, I may only want to know if a given recipient has attestations for that schema that were attested to by the coinbase attester and not the gitcoin attester.